BartoszStoppel/ECE461-Team7
# rest api with GET

did move around some stuff 
- main() is now cli() so all the router stuff doesn't get mixed in with the metric stuff
- works the same with ./run build and ./run main/test.txt
- global variables for the json bc easier to access for the router functions

to test
- ./run build and ./run main/test.txt in one terminal
- curl http://localhost:8080/repos in another terminal
- second terminal should have a json object returned to it with the score info from the urls from main/test.txt
